src/main/resources/peer-review-transform.xsl

```diff
@@ -33,29 +33,32 @@
       </p>
     </div>
   </xsl:template>
-  
+
   <xsl:template match="revision">
+    <xsl:variable name="revision-row">
```

Comment:
I gotta admit I have no idea how this works as a substitute revision counter.  I'll have to look it up.

Reply:
looked up `xsl:number`, and now I know.  :)

Reply:
Actually, the position() code looked right, but I never saw the header row (row 1). when put in some debug values, I saw the revision #0 row had position()=2,  and revision #1 had position 4 - wtf. I read up on provision which documents that it numbers elements in a node set. I then experimented with xsl:number and got expected results. I think we could use position() but would need to modify match node set.

Reply:
Thanks for the fix!  So, the revision number we displayed must have been coincidentally working for one round of revision?

Reply:
i never saw the header row. that is, first revision tag was at position 2.